app/src/main/java/com/keylesspalace/tusky/components/conversation/ConversationsFragment.kt

```diff
@@ -172,6 +179,53 @@ class ConversationsFragment : SFragment(), StatusActionListener, Injectable {
         }
     }
 
+    private fun jumpToTop() {
+        if (isAdded) {
+            layoutManager?.scrollToPosition(0)
+            recyclerView.stopScroll()
+        }
+    }
+
+    override fun setMenuVisibility(menuVisible: Boolean) {
```

Comment:
I don't think this method is meant for this usecase. What about letting each fragment know its position so it cn check itself when it should jump to the top?